boards/aerotenna/ocpoc/default.cmake

```diff
@@ -1,5 +1,7 @@
 add_definitions(
 	-D__PX4_LINUX
+
+	-D__PX4_LINUX_CONSOLE
```

Comment:
If there somewhere we wouldn't want this?

Reply:
This PR won't work with SITL at all. I don't know why. Since SITL shares `__PX4_LINUX` as well, I tried to introduce `__PX4_LINUX_CONSOLE`. Also this PR won't work with QURT because I use `poll` logic.